src/components/BlogCard/README.md

```diff
@@ -0,0 +1,28 @@
+## Store Component
```

Comment:
Let's get rid of this file and replace it with the new documentation system - see any of the `<Component Name>.examples.md` files